src/Microsoft.DocAsCode.Plugins/FileAndType.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.DocAsCode.Plugins
     public sealed class FileAndType
         : IEquatable<FileAndType>
     {
-        public FileAndType(string baseDir, string file, DocumentType type, Func<string, string> pathRewriter)
```

Comment:
It is breaking change.

---
In reply to: [77530070](https://github.com/dotnet/docfx/pull/669#discussion_r77530070) [](ancestors = 77530070)